build.gradle

```diff
@@ -306,7 +306,10 @@ project(':forge') {
                 source sourceSets.main
                 source sourceSets.userdev
                 
-                args '--mod', 'forge', '--all', '--output', rootProject.file('src/generated/resources/')
+                args '--mod', 'forge', '--all', '--output', rootProject.file('src/generated/resources/'), '--validate',
+                        '--input', rootProject.file('src/main/resources/'),
+                        //TODO this isn't ideal
+                        '--input', rootProject.gradle.gradleUserHomeDir.toPath().resolve('caches/forge_gradle/minecraft_repo/versions/1.14.4/client-extra.jar').toString()
```

Comment:
you have the classpath, you dont need this.

Reply:
I can confirm that this line is necessary, at least, removing it breaks things. Perhaps there is a better way, but I don't know it.

Reply:
What I am saying, is that whatever makes it necessary needs to change to use the classpath as a extra input so that this line does not need to be specified.